site/content/docs/5.3/components/orange-navbar.md

```diff
@@ -376,6 +376,87 @@ You can add a search input into your Global header.
 </header>
 ```
 
+### With theme selector
```

Comment:
Based on the discussion we had with the designers, maybe we should talk about "mode selector" instead of "theme selector".